src/usr/local/bin/easyrule

```diff
@@ -108,7 +108,14 @@ if (($argc > 1) && !empty($argv[1])) {
 			$message = easyrule_parse_getblock($argv[2]);
 			break;
 		case 'pass':
-			$message = easyrule_parse_pass($argv[2], $argv[3], $argv[4], $argv[5], $argv[6]);
+		    if (is_string($argv[6])){
+			    $message = easyrule_parse_pass($argv[2], $argv[3], $argv[4], $argv[5], 0, $argv[6]);
+			} elseif ($argc > 7){
+			    $message = easyrule_parse_pass($argv[2], $argv[3], $argv[4], $argv[5], $argv[6], $argv[7]);
+			} else {
+			    $message = easyrule_parse_pass($argv[2], $argv[3], $argv[4], $argv[5], $argv[6]);
+			}
```

Comment:
Indent is broken

Reply:
Hello, I dont see any indent miss-alignements, maybe I don't understand the remark?
Txs.
![easyrule_indent_check](https://user-images.githubusercontent.com/246715/32374268-30a4ab10-c09d-11e7-9929-f5c1167b4ccf.png)